src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/TupleTests.cs

```diff
@@ -261,8 +261,8 @@ .maxstack  6
   IL_002f:  ldstr      ""y""
   IL_0034:  call       ""(int A, int B) Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<(int A, int B)>(string)""
   IL_0039:  ldloc.0
-  IL_003a:  stind.ref
-  IL_003b:  ret
+  IL_003a:  stobj      ""System.ValueTuple<int, int>""
```

Comment:
Could you clarify why this isn't an error in this case?

---
In reply to: [81645590](https://github.com/dotnet/roslyn/pull/14161#discussion_r81645590) [](ancestors = 81645590,81645389,81644864,81644123)